syntax = "proto3";

message TypeMessage {
    string type_name = 1;
}